articles/cost-management-billing/microsoft-customer-agreement/manage-tenants.md

```diff
@@ -19,7 +19,9 @@ The article helps you understand and manage tenants associated with your Microso
 
 A tenant is a digital representation of your organization and is primarily associated with a domain, like Microsoft.com. It's an environment managed through Azure Active Directory that enables you to assign users permissions to manage Azure resources and billing.
 
-Each tenant is distinct and separate from other tenants, yet you can allow guest users from other tenants to access your tenant to track your costs and manage billing.
+Each tenant is distinct and separate from other tenants. You can allow users from other tenants to access your billing account by 
+-	creating guest users in your tenants and assigning the appropriate billing role(s) or by
```

Comment:
Line 22, add **:** at the end of the sentence.

Lines 22 and 23:
- Upper case the first letter of the first word for each bullet list item.
- Remove `(s)` from role.
- Add a period at the end of each bullet item since each one completes a sentence.